llvm/test/tools/llvm-mca/RISCV/SiFive7/strided-load-x0.s

```diff
@@ -37,13 +37,13 @@ vle64.v v1, (a1)
 
 # CHECK:      Iterations:        1
 # CHECK-NEXT: Instructions:      26
-# CHECK-NEXT: Total Cycles:      3523
+# CHECK-NEXT: Total Cycles:      3546
```

Comment:
The `Total Cycles` increases and `Block RThroughput` also increases, but can I consider this as what we want? As we need more cycles to occupy `SiFive7VCQ` but the resources can be better utilized. But the `Throughput` seems to be decreased in llvm-mca's simulation.

Reply:
The way to look at this patch is that it more closely resembles the actual hardware. For some programs, this will mean that `Total Cycles` and `Block RThroughput` report higher on llvm-mca, and that is (hopefully) because the actual number of cycles is higher and this change brings us closer to the actual number of cycles.

The llvm-mca numbers can be higher here and lead to an actual decrease in throughput on hardware, since the old llvm-mca report was really just an incorrect estimate.